src/olympia/blocklist/tests/test_tasks.py

```diff
@@ -284,6 +284,7 @@ def test_skip_cleanup_when_no_filters_or_config_keys(self):
     def test_cleanup_records_with_matching_attachment(self):
         mlbf = MLBF.generate_from_db(self.generation_time)
         mlbf.generate_and_write_filter(BlockType.BLOCKED)
+        mlbf.generate_and_write_filter(BlockType.SOFT_BLOCKED)
```

Comment:
Why is this not changing anything?

Reply:
On current master (since the first of this set of patches) `filter_path` always returns the path to the hard filter.. never the soft.

That's why the test wasn't failing before. Now in this patch, filter-path returns the correct path.